lib/core-ext/openstruct.rb

```diff
@@ -1,5 +0,0 @@
-class OpenStruct
```

Comment:
Looking at the commit history it was created for use in the root controller, which no longer exists.  We couldn't find any other use of it in the rest of the app.